pkg/util/conn_pool.go

```diff
@@ -0,0 +1,196 @@
+/*
+Copyright 2020 The Ceph-CSI Authors.
+
```

Comment:
can we add unit testing for this file?

Reply:
Yes, I'll do something by not creating a real connection to a Ceph cluster. That way `go test` does not rely on a Ceph environment.

Reply:
Tests for the ConnPool have been added now